### DISCUSSION TITLE
Feature Request: Custom Theme Colors

### DISCUSSION THREAD

**Discussion by tuckerconnelly** [saadeghi/daisyui#2317] [Ideas / feature requests]
I read through the docs and a lot of the code--I don't think this is possible now, but forgive me if it is.

I have two themes, and I'd like to add a custom color to both (that would be required I guess on all future themes). In this case it's `base-200-translucent` (for a translucent/blurred background that looks like base-200), but it could be useful for adding other things like `primary-darker`, etc.

**Comment by saadeghi** [comment]
See this:  https://github.com/saadeghi/daisyui/discussions/653#discussioncomment-2438428
